pills/01-why-you-should-give-it-try.xml

```diff
@@ -126,25 +125,25 @@
        Nix makes no assumptions about the global state of the system.
        This has many advantages, but also some drawbacks of course.
        The core of a Nix system is the Nix store, usually
-       installed under <literal>/nix/store</literal>, and some tools to manipulate the
+       installed under <filename>/nix/store</filename>, and some tools to manipulate the
        store. In Nix there is the notion of a <emphasis>derivation</emphasis> rather than a
        package. The difference can be subtle at the beginning, so I
        will often use the words interchangeably.
      </para>
      <para>
        Derivations/packages are stored in the Nix store as follows:
-       <literal>/nix/store/<replaceable>hash-name</replaceable></literal>,
+       <filename>/nix/store/<replaceable>hash-name</replaceable></filename>,
        where the hash uniquely identifies the derivation (this isn't quite true,
        it's a little more complex), and the name is the name of
        the derivation.
      </para>
      <para>
        Let's take a bash derivation as an example:
-       <literal>/nix/store/s4zia7hhqkin1di0f187b79sa2srhv6k-bash-4.2-p45/</literal>.
-       This picks out a directory that contains <literal>bin/bash</literal>.
+       <filename>/nix/store/s4zia7hhqkin1di0f187b79sa2srhv6k-bash-4.2-p45/</filename>.
+       This is a path in the Nix store which points to a directory that contains <filename>bin/bash</filename>.
```

Comment:
It doesn't point do a directory, it _is_ a directory which  contains bin/bash.

Reply:
Fixed.